packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/synthesize-resources.ts

```diff
@@ -184,6 +195,10 @@ const lambdaTriggers = async (coreAnswers: any, context: any, previouslySaved: a
     delete coreAnswers.triggers;
   }
 
+  if (coreAnswers.authLambdaConfig && coreAnswers.authLambdaConfig === '[]') {
+    delete coreAnswers.authLambdaConfig;
```

Comment:
Is this only attached by the code on line 177? If so, you could just not set the property there.